### ISSUE TITLE
Minor typo issue in demo/PDFdisplay.py

### ISSUE THREAD

**Issue by gbarnabic** [rk700/PyMuPDF#157]
In your demo/PDFdisplay.py file line 9 says:
"Let the user select a Python file and then scroll through it."

I assume it should say:
"Let the user select a PDF file and then scroll through it."

I was trying to get it to run and noticed it.

On a separate note, I was trying to get it to run but issues with wxPython in a virtual environment prevented me from running it. I didn't want to install wxPython in my clean Python 3.6 base install which I'm sure would have worked.  I may try to rewrite it using tkinter but not sure if there are some required features of wxPython that would cause it to loose some of it's functionality.

**Comment by JorjMcKie** [comment]
1. You are perfetly right: it should say "Let the user select a **PyMuPDF-supported** file and then scroll through it."

2. It would be great to have a tkinter version. My own experience with tkinter is close to zero, so I cannot say, what obstacle might pop up. The only thing being potentially special to wxPython is that I am drawing rectangles on the bitmap area to surround the links (if option is on) and also honor them when clicked.

3. I also must confess that I am a Qt newbee, else I would have created such a version myself.

4. The issues you had usinga virtual environment: what caused them - wxPython or PyMuPDF?

**Comment by gbarnabic** [comment]
It had to do with wxPython not really working in a virtual env on my Mac.  See this: https://wiki.wxpython.org/wxPythonVirtualenvOnMac   Basically it will work but you need to install it in your base Python instance and not in the VE.  I have started disciplined myself to install only a few packages in my base Python and use VE's for all others that are required by any projects.  I have convinced myself for some reason that this keeps things "cleaner" but it could just be my OCD, lol.  PyMuPDF installed fine in the virtual env.

**Comment by JorjMcKie** [comment]
Thanks for the insight 😊 – I was a little concerned …
BTW, are you using this new wheel option?

**Comment by gbarnabic** [comment]
Yes, I downloaded it and it worked flawlessly and it seemed to complete the install in seconds.  OS 10.13.4 Python 3.6

**Comment by JorjMcKie** [comment]
Thank you for the information. This is good to know.
I presume this issue can be closed?

**Comment by gbarnabic** [comment]
Good assumption.  Yes close it.

Thanks
